src/components/EnergyConsumptionChart/index.tsx

```diff
@@ -237,26 +231,19 @@ const EnergyConsumptionChart = () => {
   } satisfies ChartData
 
   return (
-    <Box my={16}>
-      <Center w="full" mb={{ base: 4, md: 6 }}>
-        <Box
-          position="relative"
-          maxW="500px"
-          m="auto"
-          w="80vw"
-          h={{ base: "300px", md: "400px" }}
-          mb={{ base: 4, md: 0 }}
-        >
+    <div className="my-16">
+      <div className="mb-4 flex w-full items-center justify-center md:mb-6">
```

Comment:
There are some utility components built in https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/flex.tsx

We could reuse the `Center` component in here.